requirements.txt

```
@@ -0,0 +1,4 @@
+plotly==5.22.0
+polars==0.20.23
+posit-sdk==0.2.0
```

This is the latest version. You could also unpin everything. I think either would be ok.

```suggestion
posit-sdk==0.2.2
```